CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 - Allow `ChatJoinRequest` updates
+- Fix `split` parser for tuple variants with len < 2 ([issue #834](https://github.com/teloxide/teloxide/issues/834))
```

Comment:
That's the wrong changelog, the right one is at `crates/teloxide-macros/CHANGELOG.md`